ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -98,28 +174,30 @@ const BaseLocusListLabels = React.memo((
   ) : (
     <div style={containerStyle}>
       {locusListGuids.map((locusListGuid) => {
-        const panelAppConfidence = locusListConfidence && locusListConfidence[locusListGuid]
-        let { description } = locusListsByGuid[locusListGuid] || {}
-        if (panelAppConfidence) {
-          description = (
-            <div>
-              {description}
-              <br />
-              <br />
-              <b>PanelApp gene confidence: &nbsp;</b>
-              {PANEL_APP_CONFIDENCE_DESCRIPTION[panelAppConfidence]}
-            </div>
-          )
+        const locusList = locusListsByGuid[locusListGuid]
+        const panelAppDetails = panelAppDetail[locusListGuid]
+        const { name: label, description: locusListDescription, paLocusList } = locusList
+        const { description, initials, customColor } = panelAppDetails && paLocusList ? getPaProps({
+          panelAppDetails,
+          panelAppPanel: locusListDescription || '',
+          paLocusList,
+          geneSymbol,
+        }) : {
+          description: label,
+          initials: false,
+          customColor: false,
         }
+
         return (
           <GeneDetailSection
             key={locusListGuid}
             color="teal"
-            customColor={panelAppConfidence && PANEL_APP_CONFIDENCE_LEVEL_COLORS[panelAppConfidence]}
-            maxWidth="7em"
+            customColor={customColor}
+            detail={initials}
+            maxWidth="12em"
```

Comment:
so especially with the initials not counting toward the max width, this is too wide for us. I would accept `8em` or `9em`.  Its also okay if we agree to disagree here and our forks differ a little

Reply:
Ah the initials don't count that maxWidth? I thought it looked a bit wider. I agree it's too wide in some cases.

I would have liked the widest ones without initials to be the same width as the ones showing initials.

Reply:
`8em` is quite readable on my laptop screen (1680x1050). I've added this to the PR.

<img width="1680" alt="Screen Shot 2022-05-23 at 3 38 30 pm" src="https://user-images.githubusercontent.com/3320713/169750799-6cb396a6-4b25-4fc8-a0d7-ba4b300d0b53.png">